mavros/include/mavros/setpoint_mixin.hpp

```diff
@@ -45,9 +45,9 @@ class SetPositionTargetLocalNEDMixin
   void set_position_target_local_ned(
     uint32_t time_boot_ms, uint8_t coordinate_frame,
     uint16_t type_mask,
-    Eigen::Vector3d p,
-    Eigen::Vector3d v,
-    Eigen::Vector3d af,
+    Eigen::Vector3d const & p,
+    Eigen::Vector3d const & v,
+    Eigen::Vector3d const & af,
```

Comment:
What for?

Reply:
For not copying

Reply:
It's usually questionable thing, what's faster - just copy 4 float64's or access trough a pointer all fields.
To me that looks as an unneeded preliminary optimisation.

Reply:
It is a good practice to pass non-trivial types by reference.